nix/ogmios-nixos-module.nix

```diff
@@ -0,0 +1,120 @@
+# NixOS module for configuring Ogmios service.
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.ogmios;
+in
+with lib; {
+  options.services.ogmios = with types; {
+    enable = mkEnableOption "Ogmios lightweight bridge interface for cardano-node";
+
+    package = mkOption {
+      description = "Ogmios package";
+      type = package;
+      default = pkgs.ogmios;
+    };
+
+    user = mkOption {
+      description = "User to run Ogmios service as.";
+      type = str;
+      default = "ogmios";
+    };
+
+    group = mkOption {
+      description = "Group to run Ogmios service as.";
+      type = str;
+      default = "ogmios";
+    };
+
+    nodeSocket = mkOption {
+      description = "Path to cardano-node IPC socket.";
+      type = str;
```

Comment:
I think that there exist also the `path` type but I'm not sure about the differences.